### PR TITLE
test(helper): function to pass parameters via url

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

Implemented new approach for passing `parameters` via `url`. This approach allows us:
- do not load components using `iFrame`
- not typing any characters in `Knobs`
- have already prepared `component` / `element` in proper `condition` / `state`
- only make an `assertion` using `cypress`
- decrease run time for tests: 
-- `Alert` previously took approx. `02:26` and now -> `00:54`;
-- `Badge` previously took approx. `01:00` and now -> `00:23`;

It is only the `POC` for new `approach` and I didn't do a major refactor for all components. Just made changes for 2 components to compare timing and discuss.

### Current behaviour

Currently we are passing all `parameters` using `Knobs`. Storybook takes a lot of time to be fully loaded and sometimes cypress fails to pass all `params` properly, that's why `cypress` tests often take few minutes.

### Checklist

- [ ] Screenshots are included in the PR
- [ ] Carbon implementation and Design System documentation are congruent
- [ ] All themes are supported
- [x] Commits follow our style guide
- [ ] Unit tests added or updated
- [x] Cypress automation tests added or updated
- [ ] Storybook added or updated
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/features/regression/alertActions.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Feature: Alert component
+  I want to change Alert component properties
```
Comment (KatarzynaQA) [inline cypress/features/regression/alertActions.feature:2]:
```suggestion
  I want to check Alert component actions
```

File: cypress/features/regression/test/badge.feature [hunk 2]
```diff
@@ -1,45 +1,44 @@
...
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | counter                 | nameOfObject            |
+      | 0                       | counter0                |
+      | -12                     | counter-12              |
+      | test                    | counterTest             |
+      | !@$^*()_%2B-=~[];:.,?{} | counterSpecialCharacter |
 
   @positive
   Scenario: Hover Badge component and verify that cross icon appears
+    Given I open basic Test "Badge" component page
```
Comment (KatarzynaQA) [inline cypress/features/regression/test/badge.feature:35]: move this TC to separate feature file as you did with `alert` component , please

File: cypress/features/regression/test/badge.feature [hunk 3]
```diff
@@ -1,45 +1,44 @@
...
     When I focus onto Badge component
     Then icon on preview is "cross"
 
   @positive
   Scenario: Click event
```
Comment (KatarzynaQA) [inline cypress/features/regression/test/badge.feature:40]: move this TC to separate feature file as you did with `alert` component , please

File: cypress/support/helper.js [hunk 4]
```diff
@@ -30,6 +30,17 @@ export function visitComponentUrlByThemeKnobsStory(component, theme, sufix = '',
   cy.visit(`${prepareUrl(component, 'knobs', true, prefix)}&theme=${theme}${sufix}`);
 }
 
+export function visitComponentUrlOneFewParameters(component, story, sufix = '', prefix = '', json = '', path = '', nameOfObject = '') {
```
Comment (KatarzynaQA) [inline cypress/support/helper.js:33]:
```suggestion
export function visitComponentUrlWithParameters(component, story, sufix = '', prefix = '', json = '', path = '', nameOfObject = '') {
```

File: cypress/support/step-definitions/common-steps.js [hunk 5]
```diff
@@ -32,6 +35,14 @@ Given('I open design systems {word} {string} component in no iframe', (type, com
   visitComponentUrl(component, type, true, 'design-system-');
 });
 
+Given('I open Test {word} {string} component in noIFrame with {string} json from {string} using {string} object name', (type, component, json, path, nameOfObject) => {
+  visitComponentUrlOneFewParameters(component, type, true, 'test-', json, path, nameOfObject);
```
Comment (KatarzynaQA) [inline cypress/support/step-definitions/common-steps.js:39]:
```suggestion
  visitComponentUrlWithParameters(component, type, true, 'test-', json, path, nameOfObject);
```

File: cypress/support/step-definitions/common-steps.js [hunk 6]
```diff
@@ -32,6 +35,14 @@ Given('I open design systems {word} {string} component in no iframe', (type, com
...
+});
+
+Given('I open {word} {string} component in noIFrame with {string} json from {string} using {string} object name', (type, component, json, path, nameOfObject) => {
+  visitComponentUrlOneFewParameters(component, type, true, '', json, path, nameOfObject);
```
Comment (KatarzynaQA) [inline cypress/support/step-definitions/common-steps.js:43]:
```suggestion
  visitComponentUrlWithParameters(component, type, true, '', json, path, nameOfObject);
```

File: cypress/features/regression/alertActions.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/alertActions.feature]: fixed

File: cypress/features/regression/test/badge.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/features/regression/test/badge.feature]: fixed

File: cypress/features/regression/test/badge.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/regression/test/badge.feature]: fixed

File: cypress/support/helper.js (hunk 4 above)
Comment (DlgSHi) [reply cypress/support/helper.js]: fixed

File: cypress/support/step-definitions/common-steps.js (hunk 5 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/common-steps.js]: fixed

File: cypress/support/step-definitions/common-steps.js (hunk 6 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/common-steps.js]: fixed

**Pull request merged**